recipes/openmeeg/bld.bat

```diff
@@ -0,0 +1,20 @@
+set CMAKE_CONFIG=Release
+
+mkdir build_%CMAKE_CONFIG%
+pushd build_%CMAKE_CONFIG%
+
+cmake -G "NMake Makefiles"                           ^
```

Comment:
it would be good to change this part to :
```
cmake -G "%CMAKE_GENERATOR%"  ^
```

Reply:
No, NMake is better than Visual Studio generator

Reply:
Better in what way?

Reply:
Usually faster. Visual studio generator generates solution files for visual studio that we really don't need here. NMake is lightweight. What's better would be Ninja which does compilation parallelly.

Reply:
Got it! Thanks for the explanation! :)